build/ci/production-values.yaml

```diff
@@ -85,51 +80,33 @@ varnish:
     between_bytes_timeout: 60  # Default 60s
   secrets:
     varnish:
-      ref: varnish-secret-production
+      ref: varnish-secret
```

Comment:
have we created this? IIRC it just needs to be a seed?

Reply:
It's not in the repo, so think it still needs creating